### DISCUSSION TITLE
UX: Closing the window should close settings and command bar

### DISCUSSION THREAD

**Discussion by heyivanvilla** [intellibar/main#9] [Ideas]
After I edit a setting I would quickly close the window. 2 minutes later if I open the app again instead of taking me to a blank chat it loads up the settings (or command menu).

I would prefer closing window also closes the options this way when I come back I can go straight to typing into the window.

**Comment by astoilkov** [comment]
Yeah, makes sense. I will try to fit it in the next release.
